sal_interfaces/AdamSensors/AdamSensors_Telemetry.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+    <SALTelemetry>
+        <Subsystem>AdamSensors</Subsystem>
+        <Version>3.8.2</Version>
+        <Description>Topic for telemetry from temperature sensors connected to an ADAM 6024</Description>
```

Comment:
"Topic for telemetry" is redundant. 
"Temperature readings from sensors connected to an ADAM-6024" is fine.